instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py

```diff
@@ -241,24 +281,51 @@ def _instrumented_open_call(
                     exception = exc
                     result = getattr(exc, "file", None)
                 finally:
-                    elapsed_time = round((default_timer() - start_time) * 1000)
-
+                    duration_s = default_timer() - start_time
+            response_size = 0
             if result is not None:
+                response_size = int(result.headers.get("Content-Length", 0))
                 code_ = result.getcode()
-                labels[SpanAttributes.HTTP_STATUS_CODE] = str(code_)
+                # set http status code based on semconv
+                if code_:
+                    _set_status_code_attribute(
+                        span, code_, labels, sem_conv_opt_in_mode
+                    )
 
-                if span.is_recording() and code_ is not None:
-                    span.set_attribute(SpanAttributes.HTTP_STATUS_CODE, code_)
-                    span.set_status(Status(http_status_to_status_code(code_)))
+                if _report_old(sem_conv_opt_in_mode):
+                    ver_ = str(getattr(result, "version", ""))
+                    if ver_:
+                        labels[SpanAttributes.HTTP_FLAVOR] = (
```

Comment:
I understand we are changing that because in v1.11.0 there's a need to have this for duration. Maybe a comment that isn't required anymore in new semconv?

Reply:
Good catch, added.